benchmarks/pvc/benchmarks.hpp

```diff
@@ -86,40 +67,22 @@ using PvcGemmBF16BF16FP32_RRR_4 = cutlass::gemm::device::GemmConfiguration<
         cutlass::bfloat16_t, cutlass::layout::RowMajor,
         float, cutlass::layout::RowMajor,
         float, Shape<_128, _256, _16>,
-        TiledMMA<MMAAtom, 
-                 Layout<Shape<_4,_8,_1>, Stride<_8,_1,_0>>, 
-                 Tile<Layout<Shape<_8, _4, _4>, Stride<_1, _32, _8>>,
-                      Layout<Shape<_16, _8, _1>, Stride<_1, _16, _0>>, 
-                      _16>>,
-        XE_2D_U16x32x16_LD_N, XE_2D_U16x16x16_LD_V,
         Scheduler::Gemm>;
 
-using PvcGemmBF16BF16FP32_RRR_5 = cutlass::gemm::device::GemmConfiguration<
-        cutlass::arch::IntelPVC,
-        cutlass::bfloat16_t, cutlass::layout::RowMajor,
-        cutlass::bfloat16_t, cutlass::layout::RowMajor,
-        float, cutlass::layout::RowMajor,
-        float, Shape<_8, _128, _32>,
-        TiledMMA<MMAAtom, 
-                 Layout<Shape<_1,_4,_1>, Stride<_0,_1,_0>>, 
-                 Tile<Layout<Shape<_8, _1, _1>, Stride<_1, _0, _0>>,
-                      Layout<Shape<_16, _4, _2>, Stride<_1, _32, _16>>, 
-                      _32>>,
-        XE_2D_U16x8x32_LD_N, XE_2D_U16x32x32_LD_V,
-        Scheduler::Gemm>;
+//using PvcGemmBF16BF16FP32_RRR_5 = cutlass::gemm::device::GemmConfiguration<
```

Comment:
This benchmark is currently broken. https://github.com/codeplaysoftware/cutlass-fork/blob/3162e9c4f198a9ead268aac6d82ca60255b95810/benchmarks/pvc/input.in#L21